products/sle/main.pm

```diff
@@ -1248,6 +1245,10 @@ elsif (get_var("FILESYSTEM_TEST")) {
     }
     load_filesystem_tests();
 }
+elsif (snapper_is_applicable() && get_var("BOOT_TO_SNAPSHOT") && (!(check_var("ARCH", "s390x") or (check_var('VIRSH_VMM_FAMILY', 'xen') and check_var('VIRSH_VMM_TYPE', 'linux'))))) {
```

Comment:
too long to be understandable. Better extract the whole line into `rollback_is_applicable` with better comments there why the individual flags apply. The comment line that was above the previous if applies for why we can't execute that test on s390x or xen and such.

Please also take a look into `load_rollback_tests` if something should be shared. It can be misleading to see `load_rollback_tests` and you are doing for what that name would also fit but it's not used there. Maybe move your parts into there as well?